infrastructure/virtiofs/virtiofs.go

```diff
@@ -0,0 +1,45 @@
+package virtiofs
```

Comment:
The "infrastructure" package is normally used to provide a concrete implementation of a service (a.k.a an **adapter**) that is attached via a **port** to the core application. This is from hexagonal architecture: https://medium.com/ssense-tech/hexagonal-architecture-there-are-always-two-sides-to-every-story-bc0780ed7d9c

For example, there is the [MicroVMService](https://github.com/liquidmetal-dev/flintlock/blob/main/core/ports/services.go#L12) port and the core application uses this. It not aware of the actual infrastructure. We then have to 2 adapters (i.e. Firecracker and CloudHypervisor) that can plug into this port, [both of which are defined](https://github.com/liquidmetal-dev/flintlock/tree/main/infrastructure/microvm) in the infrastructure package.

I'm wondering if there is a way that we can split this up into port & adapter? Would we want to mock virtiofs at all?